x/programs/examples/imports/pstate/pstate.go

```diff
@@ -40,7 +40,7 @@ func (*Import) Name() string {
 	return Name
 }
 
-func (i *Import) Register(link *host.Link) error {
+func (i *Import) Register(link *host.Link, _ program.Context) error {
```

Comment:
🤔 I'm not sure why/how this was going to be used.

Reply:
I might've missed something, but where's the other `Register` implementation?